### PR TITLE
Improve package build

### PR DESCRIPTION
**Pull request by dappnodedev**
Package build has been improved by using the official docker image for Prysm, instead of building from source

Also, fixed fee recipient env in setup wizard and updated github actions versions

Removed optional dependencies (as auth token has not been updated finally)

### CODE REVIEW

File: beacon-chain/Dockerfile [hunk 1]
```diff
@@ -1,30 +1,25 @@
-FROM debian:bullseye-slim as binary
-
-ARG TARGETARCH
 ARG UPSTREAM_VERSION
-ENV DOWNLOAD_URL https://github.com/prysmaticlabs/prysm/releases/download
+FROM debian:bullseye AS builder
```
Comment (Marketen) [inline beacon-chain/Dockerfile:2]: why change from `bullseye-slim` to `bullseye`?

File: beacon-chain/entrypoint.sh [hunk 2]
```diff
@@ -1,9 +1,9 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (Marketen) [inline beacon-chain/entrypoint.sh:1]: any difference between `bash` and `sh` here?

File: validator/Dockerfile [hunk 3]
```diff
@@ -1,43 +1,15 @@
...
-
-ARG TARGETARCH
 ARG UPSTREAM_VERSION
-ENV DOWNLOAD_URL https://github.com/prysmaticlabs/prysm/releases/download
-
-RUN DEBIAN_FRONTEND=noninteractive \
-  apt update && apt install --assume-yes --no-install-recommends wget ca-certificates && \
-  wget -q ${DOWNLOAD_URL}/${UPSTREAM_VERSION}/validator-${UPSTREAM_VERSION}-linux-${TARGETARCH:-amd64} -O /usr/local/bin/validator && \
-  chmod +x  /usr/local/bin/validator
+FROM debian:bullseye AS builder
```
Comment (Marketen) [inline validator/Dockerfile:2]: same as beacon-chain, why stop using slim version?

File: beacon-chain/entrypoint.sh [hunk 4]
```diff
@@ -25,33 +25,25 @@ case $_DAPPNODE_GLOBAL_EXECUTION_CLIENT_PRATER in
...
+if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" = "true" ]; then
   echo "MEVBOOST is enabled"
   MEVBOOST_URL="http://mev-boost.mev-boost-goerli.dappnode:18550"
-  if curl --retry 5 --retry-delay 5 --retry-all-errors "${MEVBOOST_URL}"; then
-    EXTRA_OPTS="--http-mev-relay=${MEVBOOST_URL} ${EXTRA_OPTS}"
-  else
-    echo "MEVBOOST is enabled but ${MEVBOOST_URL} is not reachable"
-    curl -X POST -G 'http://my.dappnode/notification-send' --data-urlencode 'type=danger' --data-urlencode title="${MEVBOOST_URL} is not available" --data-urlencode 'body=Make sure the mevboost is available and running'
-  fi
+  EXTRA_OPTS="--http-mev-relay=${MEVBOOST_URL} ${EXTRA_OPTS}"
```
Comment (Marketen) [inline beacon-chain/entrypoint.sh:31]: what happens if mev-boost is set but unreachable?

File: beacon-chain/entrypoint.sh (hunk 2 above)
Comment (dappnodedev) [reply beacon-chain/entrypoint.sh:1]: Bash provides extended features that are not needed here

File: validator/Dockerfile (hunk 3 above)
Comment (dappnodedev) [reply validator/Dockerfile]: Same as above

File: beacon-chain/entrypoint.sh (hunk 4 above)
Comment (dappnodedev) [reply beacon-chain/entrypoint.sh:31]: Same as what would happen if you start consensus client while MEV Boost pkg is running but it stops after.

**Review by pablomendezroyo — APPROVED** [review]
Way more better!

**Pull request merged**